bedrock/security/templates/security/bug-bounty/web-eligible-sites.html

```diff
@@ -0,0 +1,190 @@
+{% extends "security/base.html" %}
+
+{% block page_title %}Mozilla Web Application Security Bug Bounty FAQ{% endblock %}
+{% set body_id = "web-eligible-sites" %}
+
+
+{% block breadcrumbs %}
+  <nav class="breadcrumbs">
+    <a href="{{ url('mozorg.home') }}">{{ _('Home') }}</a> &gt;
+    <a href="{{ url('security.index') }}">{{ _('Mozilla Security') }}</a> &gt;
+    <a href="{{ url('security.bug-bounty') }}">{{ _(' Bug Bounty ') }}</a> &gt;
+  </nav>
+{% endblock %}
+
+{% block article %}
+  <article class="section-content" itemscope itemtype="http://schema.org/Article">
+    <header>
+      <h1 itemprop="name" class="title-shadow-box">Eligible Websites &amp; Services</h1>
+    </header>
+
+    <div itemprop="articleBody">
+      <h2>
+        <strong id="critical-sites">Critical</strong>
+      </h2>
+
+      <p>These websites and services are considered critical to Mozilla operations and pay out at the highest bounty rate.</p>
+      <h4>ABSearch</h4>
+      <ul>
+        <li>search.services.mozilla.com</li>
+      </ul>
+
+      <h4>Add-ons</h4>
+      <ul>
+        <li>addons.cdn.mozilla.net</li>
+        <li>addons.mozilla.org</li>
+        <li>blocklist.addons.mozilla.org</li>
+        <li>builder.addons.mozilla.org</li>
+        <li>compatibility-lookup.services.mozilla.com</li>
+        <li>discovery.addons.mozilla.org</li>
+        <li>services.addons.mozilla.org</li>
+        <li>static.addons.mozilla.net</li>
+        <li>versioncheck-bg.addons.mozilla.org</li>
+        <li>versioncheck.addons.mozilla.org</li>
+      </ul>
+
+      <h4>Bedrock (www)</h4>
+      <ul>
+        <li>www.firefox.com</li>
+        <li>www.getfirefox.com</li>
+      	<li>www.mozilla.com</li>
+      	<li>www.mozilla.org</li>
+      </ul>
+
+      <h4>Bugzilla</h4>
+      <ul>
+        <li>bugzilla.mozilla.org</li>
+        <li>*.bmoattachments.org (excludes XSS and open redirects)</li>
+      </ul>
+      <p>
+        <strong>Please do not use automated scanners, create, or modify bugs when testing Bugzilla.</strong>
+        <br><br>Instead, install your own local copy for testing from <a href="https://github.com/mozilla/webtools-bmo-bugzilla">webtools-bmo-bugzilla</a> or use our <a href="https://bugzilla-dev.allizom.org/">development instance</a>.
```

Comment:
@dylanwh Would you like to add a bit about the vagrantfile here, for "no seriously it's that easy" opportunity?